safe-global/web-core
# refactor: Remove old safe creation code

## What it solves

Resolves #1404 

## How this PR fixes it

- Removes old safe creation components under `/src/components/create-safe`
- Moves applicable tests to new directory `/src/components/new-safe/create`
- Removes abTest for safe creation
- Renames create safe steps to be more descriptive

## How to test it

1. Open the Safe
2. Go through the safe creation flow
3. Observe that it still works
